cms/src/main/java/uk/nhs/digital/apispecs/swagger/IsAnyTrueHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.apispecs.swagger;
+
+import com.github.jknack.handlebars.Helper;
+import com.github.jknack.handlebars.Options;
+import org.apache.commons.lang.Validate;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Optional;
+
+public class IsAnyTrueHelper implements Helper<String> {
+
+    public static final IsAnyTrueHelper INSTANCE = new IsAnyTrueHelper();
+
+    public static final String NAME = "isAnyTrue";
+
+    @Override public Object apply(final String items, final Options options) throws IOException {
+
+        Validate.notEmpty(options.params,"One or more input parameters");
```

Comment:
I'm afraid this version of the message still doesn't communicate what the failure was.

When I make this validation fail, what I'm getting in the logs is `java.lang.IllegalArgumentException: One or more input parameters` and still have to analyse source code to figure out what the error was about.

This validation requires that at least one parameter is provided.

Why not communicate this explicitly in the message, saying something like 'At least one parameter is required but none was provided.'?